### DISCUSSION TITLE
Welcome to avengers Discussions!

### DISCUSSION THREAD

**Discussion by edincodes** [edincodes/avengers#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edinvd** [comment]
Primer Discussions abierto 😆 

Estamos avanzando

**Reply by iamedinv** [reply]
Es un buen momento 🤖 @edincodes @edinvd

**Reply by edincodes** [reply]
Definitivamente @iamedinv 🚀

**Comment by iamedinv** [comment]
Aun esta en Beta, pero se ve que es una muy buena implementación. Es una forma mas directa de poder discutir cosas sobre el proyecto
